src/supervisor/watchers/handlers/pod.ts

```diff
@@ -120,6 +124,22 @@ export function isPodReady(pod: V1Pod): boolean {
   );
 }
 
+export async function paginatedPodList(namespace: string): Promise<{
+  response: IncomingMessage;
+  body: V1PodList;
+}> {
+  const v1PodList = new V1PodList();
+  v1PodList.apiVersion = 'v1';
+  v1PodList.kind = 'PodList';
+  v1PodList.items = new Array<V1Pod>();
+
+  return await paginatedList(
+    namespace,
+    v1PodList,
+    k8sApi.coreClient.listNamespacedPod.bind(k8sApi.coreClient),
```

Comment:
what does this line do?

Reply:
Just silencing the TS compiler/linter

`bind` ensures that if `listNamespacedPod` references `this` in its implementation, that `this` returns the right value